Description: Add missing wctype.h
Author: Walter Doekes (work) <wjdoekes@osso.nl>
Forwarded: no
Last-Update: 2018-02-27

--- maodbc-3.0.3.orig/ma_debug.c
+++ maodbc-3.0.3/ma_debug.c
@@ -19,6 +19,8 @@
 #include <ma_odbc.h>
 
 #ifdef MAODBC_DEBUG
+#include <stdio.h>
+#include <wchar.h>
 extern char LogFile[];
 
 void ma_debug_print(my_bool ident, char *format, ...)
--- maodbc-3.0.3.orig/ma_odbc.h
+++ maodbc-3.0.3/ma_odbc.h
@@ -44,6 +44,7 @@
 #include <stddef.h>
 #include <assert.h>
 #include <time.h>
+#include <wctype.h>
 
 #include <ma_odbc_version.h>
 
